src/scripts/main.js

```diff
@@ -1,3 +1,37 @@
 'use strict';
 
-// write code here
+const worker = document.querySelector('ul');
+const personList = [...worker.children];
+
+function sortSalary(list) {
+  const sortedEmployees = list.sort((a, b) => {
+    return (convertToNum(b.dataset.salary)
+      - convertToNum(a.dataset.salary));
+  });
+
+  for (const employee of sortedEmployees) {
+    worker.insertAdjacentElement('beforeend', employee);
+  }
+}
+
+function getEmployees(list) {
+  const employees = list.map(elem => {
+    return {
+      name: elem.innerText,
+      position: elem.dataset.position,
+      salary: elem.dataset.salary,
+      age: elem.dataset.age,
```

Comment:
you can use spread here
```suggestion
      ...elem.dataset,
```